### ISSUE TITLE
Environmental disturbance on dolphin populations mediated through food networks

### ISSUE THREAD

**Issue by aebudden** [NCEAS/oss-2017#20]
Author: **Tingting Tang**
Topic: Cetacean populations

## Summary of the synthesis topic
We develop a stage structured population model for bottlenose dolphin in Gulf of Mexico to study the potential impact of an environmental disturbance on their population through food network effect. 

## Data needs
Life table, life history, growth phase data, stranding data in Gulf of Mexico coastal and abundance estimation available on bottlenose dolphin. Main diet source of bottlenose dolphin, such as fish and squid, and their abundance and seasonal trend in Gulf of Mexico. 

## Analytical approaches
First, using data available on the bottlenose dolphin life history to set up the base for a discrete stage structured population model. Then, combining the data of food consumption habit of the bottlenose dolphin with the abundance trend of both the dolphins and their food source to develop a model framework, which specify the relation between the food source abundance with the vital rates of bottlenose dolphin from each life stage. Next, employing the seasonal trend of the abundance of the main food source to construct the difference equation describing their population dynamic. With the above steps, a discrete stage structured population model for bottlenose dolphin coupled with their prey population is established. Using this model, we can symbolically calculate the equilibrium of the bottlenose dolphin and investigate the stability of the equilibrium. Next, to parameterize the model, one can utilize the stranding data of bottlenose dolphin in Gulf of Mexico coastal and available life table data. we can assume a range of values for those parameters which maybe impossible to estimate based on currently available data. Then perform sensitivity analysis of all parameters of interest.

## Impacts of the synthesis
We show how to utilize established theory on discrete population models to extract useful insights from large and scattered data. We demonstrate that mathematical modeling is a very useful and economically efficient means to obtain insights of the population dynamics of the bottlenose dolphin under the influence of an environmental disturbance.

**Comment by kfrasier** [comment]
How does this compare to the recent paper by Schwacke et al*? 
I think they did something similar, but had to resort to a lot of expert interviews to estimate all of the life history parameters.  

* Schwacke, et al. (2017). Quantifying injury to common bottlenose dolphins from the Deepwater Horizon oil spill using an age-, sex-and class-structured population model. Endangered Species Research, 33, 265-279.

**Comment by tttang0602** [comment]
@kfrasier Yeah. It is a great paper on the trajectory of where I want to go. They used an age structured model to study the impact from the oil spill on the dolphins. I am thinking to use an stage structured model rather than age structured and I want to add an extra dimension of impact through the food web.  I actually worked on a paper recently published focusing on developing a general stage structured model to study how large marine mammals respond to disturbances such as oil spills. 

Analysis of lethal and sublethal impacts of environmental disasters on sperm whales using stochastic modeling.   https://link.springer.com/article/10.1007/s10646-017-1813-4

**Comment by paulacetus** [comment]
I’m also interested in food network effects. While I’m also very interested in bottlenose dolphins and think it’s a great case study as far as the life history component (because for the most part information is available), I’m wondering if it is a good choice as far as analyzing food web effects as proposed in this study. BDs are opportunistic species that feed on a variety of marine species, and have developed several ways to minimize foraging effort, including feeding from shrimp vessels. You would expect BDs to switch to the most available species. I guess some assumptions could be made in the prey model, but I’m not sure there is much knowledge regarding specific prey. Unless you are considering a much more general approach (e.g. prey groups). Could you provide more details about the prey model? Another issue is that the impacts on populations are likely to be very site-dependent. The proposal mentions using the coastal BD population for the life history. Are you considering coastal or gulf-wide food web effects? I’m interested in this approach, so I’m trying to understand its feasibility.

**Comment by paulacetus** [comment]
I wonder if we could go for an "hybrid" with my proposal (#18 Dolphin-shrimp interactions) and instead of using prey, use shrimpers, which are active throughout the Gulf. There are fisheries data available online for this purpose. We could piggy-back on other proposals that gather fisheries data. We would still have assumptions in terms of the effects of foraging on the population parameters, but perhaps would need fewer.

**Comment by kfrasier** [comment]
@tttang0602 Got it - thanks for the paper link! I'd definitely be interested in learning the nuts and bolts of how these models work. The prey consumption/food web pressure question has also been on my mind for awhile for other species, but coastal bottlenose would be an interesting case study.

**Comment by tttang0602** [comment]
@paulacetus I think that is a great idea! The available shrimpers data could be incorporated in the model by adjusting the parameters. So far, I am thinking to follow the paper below for the prey model. But more data could definitely help to develop a more bottlenose dolphin specific model. Depends on what we want to understand and analyze,  we can adjust the model even further.

SOPHIA R.-J. JANG and AZMY S. ACKLEH, Discrete-time, discrete stage-structured predator–prey models. Journal of Difference Equations and Applications, Vol. 11, No. 4–5, April 2005, 399–413.
